tests/filecheck/dialects/riscv/riscv_ops_custom_syntax.mlir

```diff
@@ -0,0 +1,270 @@
+  // CHECK-NEXT: %lw = riscv.lw %0 1 : !riscv.reg<>
+  // CHECK-NEXT: riscv.sb %0, %1 1 :
+  // CHECK-NEXT: riscv.sh %0, %1 1 :
+  // CHECK-NEXT: riscv.sw %0, %1 1 :
+
+  // RV32I/RV64I: 2.8 Control and Status Register Instructions
+
+  %csrrw_rw = "riscv.csrrw"(%0) {"csr" = 1024 : i32}: (!riscv.reg<>) -> !riscv.reg<>
+  %csrrw_w = "riscv.csrrw"(%0) {"csr" = 1024 : i32, "writeonly"}: (!riscv.reg<>) -> !riscv.reg<>
+  %csrrs_rw = "riscv.csrrs"(%0) {"csr" = 1024 : i32}: (!riscv.reg<>) -> !riscv.reg<>
+  %csrrs_r = "riscv.csrrs"(%0) {"csr" = 1024 : i32, "readonly"}: (!riscv.reg<>) -> !riscv.reg<>
+  %csrrc_rw = "riscv.csrrc"(%0) {"csr" = 1024 : i32}: (!riscv.reg<>) -> !riscv.reg<>
+  %csrrc_r = "riscv.csrrc"(%0) {"csr" = 1024 : i32, "readonly"}: (!riscv.reg<>) -> !riscv.reg<>
+  %csrrsi_rw = "riscv.csrrsi"() {"csr" = 1024 : i32, "immediate" = 8 : i32}: () -> !riscv.reg<>
+  %csrrsi_r = "riscv.csrrsi"() {"csr" = 1024 : i32, "immediate" = 0 : i32}: () -> !riscv.reg<>
+  %csrrci_rw = "riscv.csrrci"() {"csr" = 1024 : i32, "immediate" = 8 : i32}: () -> !riscv.reg<>
+  %csrrci_r = "riscv.csrrci"() {"csr" = 1024 : i32, "immediate" = 0 : i32}: () -> !riscv.reg<>
+  %csrrwi_rw = "riscv.csrrwi"() {"csr" = 1024 : i32, "immediate" = 1 : i32}: () -> !riscv.reg<>
+  %csrrwi_w = "riscv.csrrwi"() {"csr" = 1024 : i32, "writeonly", "immediate" = 1 : i32}: () -> !riscv.reg<>
+  // CHECK-NEXT: %csrrw_rw = riscv.csrrw %0 1024 : !riscv.reg<>
+  // CHECK-NEXT: %csrrw_w = riscv.csrrw %0 1024 w : !riscv.reg<>
+  // CHECK-NEXT: %csrrs_rw = riscv.csrrs %0 1024 : !riscv.reg<>
+  // CHECK-NEXT: %csrrs_r = riscv.csrrs %0 1024 r : !riscv.reg<>
+  // CHECK-NEXT: %csrrc_rw = riscv.csrrc %0 1024 : !riscv.reg<>
+  // CHECK-NEXT: %csrrc_r = riscv.csrrc %0 1024 r : !riscv.reg<>
+  // CHECK-NEXT: %csrrsi_rw = riscv.csrrsi 1024 8 : !riscv.reg<>
+  // CHECK-NEXT: %csrrsi_r = riscv.csrrsi 1024 0 : !riscv.reg<>
+  // CHECK-NEXT: %csrrci_rw = riscv.csrrci 1024 8 : !riscv.reg<>
+  // CHECK-NEXT: %csrrci_r = riscv.csrrci 1024 0 : !riscv.reg<>
+  // CHECK-NEXT: %csrrwi_rw = riscv.csrrwi 1024 1 : !riscv.reg<>
+  // CHECK-NEXT: %csrrwi_w = riscv.csrrwi 1024 w 1 : !riscv.reg<>
+
+  // Machine Mode Privileged Instructions
+  "riscv.wfi"() : () -> ()
+  // CHECK-NEXT: riscv.wfi :
+
+
+  // RV32M/RV64M: 7 “M” Standard Extension for Integer Multiplication and Division
+
+  // Multiplication Operations
+  %mul = "riscv.mul"(%0, %1) : (!riscv.reg<>, !riscv.reg<>) -> !riscv.reg<>
+  %mulh = "riscv.mulh"(%0, %1) : (!riscv.reg<>, !riscv.reg<>) -> !riscv.reg<>
+  %mulhsu = "riscv.mulhsu"(%0, %1) : (!riscv.reg<>, !riscv.reg<>) -> !riscv.reg<>
+  %mulhu = "riscv.mulhu"(%0, %1) : (!riscv.reg<>, !riscv.reg<>) -> !riscv.reg<>
+  // CHECK-NEXT: %mul = riscv.mul %0, %1 : !riscv.reg<>
+  // CHECK-NEXT: %mulh = riscv.mulh %0, %1 : !riscv.reg<>
+  // CHECK-NEXT: %mulhsu = riscv.mulhsu %0, %1 : !riscv.reg<>
+  // CHECK-NEXT: %mulhu = riscv.mulhu %0, %1 : !riscv.reg<>
```

Comment:
We should stick to either having commas between arguments or not, but this mix feels inconsistent

Reply:
For example, here you don't have commas between arguments: https://github.com/xdslproject/xdsl/pull/1205/files#diff-e6f0eda1be37e0ffda7153c54d1b1b5fa6ecb41e84399d242a82721a6391eac3R22

Reply:
For consistency sake I would go either always with or without commas (so both for attributes and ssa arguments).

Reply:
That way we would have:

```
%res = riscv.add %a, %b
%res2 = riscv.addi %a, 4
```

And they maintain a consistent format.

Reply:
Updated.